Work with older versions of CMake, such as CMake version 3.11.4 on CentOS 8 and
CMake version 3.10.2 on Ubuntu 18.04 Bionic.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,11 +1,10 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.9)

 project(
     simdjson
     # The version number is modified by tools/release.py
     VERSION 1.0.0
     DESCRIPTION "Parsing gigabytes of JSON per second"
-    HOMEPAGE_URL "https://simdjson.org/"
     LANGUAGES CXX C
 )

@@ -123,23 +122,8 @@ install(
     COMPONENT simdjson_Development
 )

-install(
-    TARGETS simdjson
-    EXPORT simdjsonTargets
-    RUNTIME COMPONENT simdjson_Runtime
-    LIBRARY COMPONENT simdjson_Runtime
-    NAMELINK_COMPONENT simdjson_Development
-    ARCHIVE COMPONENT simdjson_Development
-    INCLUDES DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
-)
-
 configure_file(cmake/simdjson-config.cmake.in simdjson-config.cmake @ONLY)

-write_basic_package_version_file(
-    simdjson-config-version.cmake
-    COMPATIBILITY SameMinorVersion
-)
-
 set(
     SIMDJSON_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/simdjson"
     CACHE STRING "CMake package config location relative to the install prefix"
@@ -154,13 +138,6 @@ install(
     COMPONENT simdjson_Development
 )

-install(
-    EXPORT simdjsonTargets
-    NAMESPACE simdjson::
-    DESTINATION "${SIMDJSON_INSTALL_CMAKEDIR}"
-    COMPONENT example_Development
-)
-
 #
 # CPack
 #
